README.md

```diff
@@ -101,3 +91,19 @@ repo.getIssues(false).ifPresent(issueData -> issueData.forEach(issue -> {
         System.out.println(comment.user.username + ": " + comment.body));
 }));
 ```
+
+### Further data processing
+
+The data extracted by this tool can be further processed, for example using the `run-issues.py` skript from the tool [`codeface-extraction`](https://github.com/se-sic/codeface-extraction). This organises and unifies the issue data into a single .list file. It also allows for synchronisation with data from other data extraction tools, such as `codeface`.
```

Comment:
Two comments here:
(1) ... into a single csv-like .list file (nobody knows here what a .list file is, so we should refer to csv)
(2) Could you please use American English here for consistency reasons? means: organizes, synchronization